README.md

```diff
@@ -8,4 +8,245 @@ are in the process of being changed and will certainly break your code.
+                                      .UpsertString( key2, "bar value")
+                                      .Build()
+    ctx2 := tags.ContextWithChanges(ctx, newTagSet)
+
+## Registering views and retrieving their collected data.
+
+### To create/retrieve/delete a measure a.k.a resource
+Create/load measures units:
+
+    // returns a *MeasureFloat64
+    mf, err := stats.NewMeasureFloat64("/my/float64/measureName", "some measure")
+    if err != nil {
+        // handle error
+    }
+    mi, err := stats.NewMeasureInt64("/my/otherName", "some other measure")
+    if err != nil {
+        // handle error
+    }    
+    ...
+
+Retrieve measure by name:
+    mf, err := stats.GetMeasureByName("my/float64/measureName")
+	if err != nil {
+        // handle error
+    }
+    mi, err := stats.GetMeasureByName("/my/otherName")
+    if err != nil {
+        // handle error
+    }
+    ...
+
+Delete measure (this can be useful when replacing a measure by another measure with the same name):
+	if err := stats.DeleteMeasure(mf); err != nil {
+        // handle error
+    }
+    if err := stats.DeleteMeasure(mi); err != nil {
+        // handle error
+    }
+    ...
+
+### To create an aggregation type
+Currently only 2 types of aggregations are supported. The AggregationCount is used to count the number of times a sample was recorded. The AggregationDistribution is used to provide a histogram of the values of the samples.
+
+    histogramBounds := []float64 { -10, 0, 10, 20}
+    agg1 := stats.NewAggregationDistribution(histogramBounds)
+    agg2 := stats.NewAggregationCount()
+
+
+### To create an aggregation window
+Currently only 3 types of aggregation windows are supported. The WindowCumulative is used to continuously aggregate the data received. The WindowSlidingTime to aggregate the data received over the last specified time interval. The NewWindowSlidingCount to aggregate the data received over the last specified sample count.
+Currently all aggregation types are compatible with all aggregation windows. Later we might provide aggregation types that are incompatible with some windows.
+
+    duration := 10 * time.Second
+    precisionIntervals := 5
+    wnd1 := stats.NewWindowSlidingTime(duration, precisionIntervals)
+
+    lastNSamples := 100
+    precisionSubsets := 10
+    wnd2 := stats.NewWindowSlidingCount(lastNSamples, precisionSubsets)
+    
+    wn3 := stats.NewWindowCumulative()
+
+
+### To creater/register a view
+Create a view:
+
+    vw1 = stats.NewViewInt64("my/int64/viewName", "some description", []Keys{key1, key2}, mf, agg1, wnd1)
+    vw2 := stats.NewViewFloat64("my/float64/viewName", "some other description", []Keys{key1}, mi, agg2, wnd3)
+
+Register view:
+
+    if err := stats.RegisterView(vw1); err != nil {
+        // handle error
+    }
+    if err := stats.RegisterView(vw2); err != nil {
+        // handle error
+    }
+
+Retrieve view by name:
+    vw1, err := stats.GetViewByName("my/int64/viewName")
```

Comment:
I'd name this with a more representative name, e.g. countView := ...

Reply:
Just called it myView